inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -538,6 +538,196 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
+    });
+
+    addSpecificCreator({"PriorBoxClusteredIE"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "PriorBoxClustered",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<CNNLayer>(attrs);
+        res->params = params;
+        res->params["clip"] =
+            res->getBoolStrParamAsIntStr("clip");
+
+        auto step_h = std::stof(res->params["step_h"]);
+        auto step_w = std::stof(res->params["step_w"]);
+        if (std::abs(step_h - step_w) < 1e-5) {
+            res->params["step"] = res->params["step_w"];
+        }
+        return res;
+    });
+
+    addSpecificCreator({"ProposalIE"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Proposal",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<CNNLayer>(attrs);
+        res->params = params;
+        res->params["clip_before_nms"] =
+            res->getBoolStrParamAsIntStr("clip_before_nms");
+        res->params["clip_after_nms"] =
+            res->getBoolStrParamAsIntStr("clip_after_nms");
+        res->params["normalize"] = res->getBoolStrParamAsIntStr("normalize");
+        return res;
+    });
+
+    addSpecificCreator({"Relu"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "ReLU",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<ReLULayer>(attrs);
+        res->params = params;
+        return res;
+    });
+
+    addSpecificCreator({"Reshape"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Reshape",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<ReshapeLayer>(attrs);
+        return res;
+    });
+
+    addSpecificCreator({"ReverseSequence"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "ReverseSequence",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<ReverseSequenceLayer>(attrs);
+        res->params = params;
+        return res;
+    });
+
+    addSpecificCreator({"ROIPooling"},
```

Comment:
I don't think that we need specific creator for ROIPooling. We don't change the type or any attributes, it should work from the box.